draft-ietf-httpbis-cache-latest.xml

```diff
@@ -1613,14 +1615,22 @@
 <t>
    The "public" response directive indicates that a cache &MAY; store the
    response even if it would otherwise be prohibited, subject to the
-   constraints of any other response directives present. In other words,
+   constraints defined in <xref target="response.cacheability"/>. In other words,
    public explicitly marks the response as cacheable. For example,
    public permits a shared cache to reuse a response to a request containing
    an Authorization header field (<xref target="caching.authenticated.responses"/>).
 </t>
 <t>
-   If no explicit freshness information is provided, the response is
-   is heuristically cacheable (<xref target="heuristic.freshness"/>).
+   Note that it is not necessary to add public to a response if the response
+   is explicitly fresh (e.g., due to the presence of the max-age response
+   directive or the Expires header), or if the response is heuristically
+   cacheable (e.g., the 200 and several other response status codes; see
+   <xref target="heuristic.freshness"/>).
```

Comment:
But this paragraph is contradicting the sentence above it (an example where neither max-age nor Expires is sufficient to allow a shared cache to cache the response -- only s-max-age would). Can we combine those examples into one paragraph that shows when public is useful and when it is not?

Also, this paragraph fails to account for extension cache directives, whereas the one above does because of the reference to response.cacheability.

Honestly, I still see no reason to prevent people from marking a response as public even if it is already public by default. Maybe they are working around some other piece of software that has different heuristics?